backend/api/tests/views/test_views_algo.py

```diff
@@ -525,6 +525,10 @@ def test_algo_create_fail(self, _):
         response = self.client.post(self.url, data={}, format="json")
         self.assertEqual(response.status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
 
+    # def test_algo_create_fail(self, _):
```

Comment:
Can this be uncommented?

Reply:
Totally removed, this was not relevant anymore. Thanks for spotting it!